src/Middleware/ExecutionChain.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+namespace League\Route\Middleware;
+
+use \Psr\Http\Message\ResponseInterface;
```

Comment:
Use statement are always from global and `\` prefixes are troublesome is more exotic php versions/envs. I'd recommend removing it, I've dealt with this in the past and it wasn't fun.